README.md

```diff
@@ -3,64 +3,91 @@
 ## Learning Goals
 
 - Define web programming
-- Identify the “Three Pillars of Web Programming"
+- Identify reference example
+- Identify the "Three Pillars of Web Programming"
 
 ## Introduction
 
-HTML, CSS and JavaScript are the main tools we use to build the web. To imagine
-how they all work together, think about the web as a robot. HTML is the robot's
-structure, CSS is the paint on the robot, plus how its pieces are positioned,
-and JavaScript is what actually makes the robot move. So if we want to build web
-applications that accept, change and return information (in other words, make
-the robot _move_), we'll need understand the Document Object Model (DOM), the
-JavaScript language and the principles of web programming.
+"Front-End Web Programming" is a phrase that gets used in a lot of places by a
```

Comment:
@jenmyers I didn't get this robot metaphor. I thought the intro to the doc was much stronger.